src/main/java/seedu/address/logic/commands/ModuleCommand.java

```diff
@@ -66,8 +67,12 @@ public CommandResult execute(Model model) throws CommandException {
             throw new CommandException(MESSAGE_PERSON_NOT_FOUND);
         }
 
-        ArrayList<Module> modules = person.getModules();
-        if (modules.contains(module)) {
+        ArrayList<String> modules = person.getModules()
+                .stream()
+                .map(x -> x.toString().toUpperCase())
+                .collect(Collectors.toCollection(ArrayList::new));
+
+        if (modules.contains(module.toString().toUpperCase())) {
```

Comment:
I think this might be redundant as the equals() method of Module already handles case insensitivity.